docs/core/app-types.md

```diff
@@ -138,3 +140,10 @@ The following `project.json` sample illustrates a simple self-contained applicat
 }
 ```
 
+### Making a smaller self-contained application
+As mentioned, self-contained application will package the entire runtime with the application. The "entire runtime" here 
```

Comment:
This section doesn't seem super useful. It describes the problem and then sends you somewhere else. Feels like we might have a factoring problem between the two documents.